<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Студенты</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>951</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateButton">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить данные</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>791</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Добавление</string>
       </attribute>
       <widget class="QPushButton" name="InsertButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>220</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Факультет</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Возраст</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="GroupLine">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>130</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Индекс</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>190</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Кафедра</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LastNameLine">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>100</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Группа</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="NameLine">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="AgeLine">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="IndexLine">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>70</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Изменение</string>
       </attribute>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Возраст</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="GroupLine_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>130</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Индекс</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>190</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Кафедра</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>220</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Факультет</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StudentChangeButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="LastNameLine_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>100</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Группа</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>271</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="NameLine_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="AgeLine_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="IndexLine_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>70</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Фамилия</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="StudentChangeID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Удаление</string>
       </attribute>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="StudentDeleteID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="StudentDeleteButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Удаление</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Кафедры</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>951</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateButton_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить данные</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>791</width>
        <height>321</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Добавление</string>
       </attribute>
       <widget class="QLineEdit" name="KafedraInsertName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="KafedraInsertButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Название кафедры</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Изменение</string>
       </attribute>
       <widget class="QLineEdit" name="KafedraChangeID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Название кафедры</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="KafedraChangeName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="KafedraChangeButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Удаление</string>
       </attribute>
       <widget class="QLineEdit" name="KafedraDeleteID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="KafedraDeleteButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Удаление</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Факультеты</string>
     </attribute>
     <widget class="QDial" name="dial_2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>10</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>10</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>80</y>
        <width>108</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>951</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="UpdateButton_3">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>330</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить данные</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>831</width>
        <height>321</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Добавление</string>
       </attribute>
       <widget class="QPushButton" name="FakultetInsertButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Название факультета</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="FakultetInsertName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>Изменение</string>
       </attribute>
       <widget class="QLabel" name="label_40">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>40</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Название факультета</string>
        </property>
       </widget>
       <widget class="QPushButton" name="FakultetChangeButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="FakultetChangeName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="FakultetChangeID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>Удаление</string>
       </attribute>
       <widget class="QLineEdit" name="FakultetDeleteID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>271</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="FakultetDeleteButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Удаление</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="blocker">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="login">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_login">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signin">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>120</y>
      <width>172</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="blocker_user">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>30</y>
      <width>1021</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>blocker_user</zorder>
   <zorder>blocker</zorder>
   <zorder>label_login</zorder>
   <zorder>label_password</zorder>
   <zorder>login</zorder>
   <zorder>password</zorder>
   <zorder>signin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
